es-ES/mail/chrome/messenger/messenger.properties

```diff
@@ -424,10 +424,12 @@ editVirtualFolderPropertiesTitle=Editar las propiedades de la búsqueda %S
 #alert to inform the user to choose one or more folders to search for a saved search folder
 alertNoSearchFoldersSelected=Debe elegir al menos una carpeta en la que buscar para guardar la búsqueda.
 
-# These two are displayed in the message and folder pane windows
-# LOCALIZATION NOTE(????ByteAbbreviation): Do not translate %d below, it is the size of the message/folder
-kiloByteAbbreviation=%dKB
-megaByteAbbreviation=%dMB
+# These are displayed in the message and folder pane windows
+# LOCALIZATION NOTE %.*f is the abbreviated size in the appropriate units
+byteAbbreviation2=%.*f bytes
```

Comment:
I don't know why %d wouldn't work. That's standard printf stuff. As long as it matches the others, it should work. Problem here is we didn't want to remove the 2 versions.

Did you test this new version and it worked?